Software/scan_key/ErrorSQL.py

```diff
@@ -7,9 +7,10 @@
 from email.mime.text import MIMEText
 
 def errorSQL(id, errMessage):
+    cnx = pymysql.connect(user='CATS', password='CATS', host='192.168.1.2', database='CATS', autocommit=True)
```

Comment:
Let's change this back to CATS-SQL.local. I've figured out a way to temporarily resolve the hostname issues.